modern/src/App.js

```diff
@@ -341,6 +341,62 @@ function Group(props) {
   );
 }
 
+function Text({
+  node,
+  id,
+  children,
+  display,
+  ticker,
+  antialias,
+  x,
+  y,
+  w,
+  h,
+  font,
+  fontsize,
+  color,
+  align,
+}) {
+  const params = {};
+  if (x !== undefined) {
+    params.left = Number(x);
+  }
+  if (y !== undefined) {
+    params.top = Number(y);
+  }
+  if (w !== undefined) {
+    params.width = Number(w);
+  }
+  if (h !== undefined) {
+    params.height = Number(h);
+  }
+  if (color !== undefined) {
+    params.color = `rgb(${color})`;
+  }
+  if (fontsize !== undefined) {
+    params.fontSize = `${fontsize}px`;
+  }
+  if (align !== undefined) {
+    params.textAlign = align;
+  }
+  return (
+    <div
+      data-node-type="Text"
+      data-node-id={id}
+      draggable={false}
+      style={{
+        position: "absolute",
+        userSelect: "none",
+        MozUserSelect: "none",
```

Comment:
Wouldn't have thought this was necessary, but Firefox was still allowing selection without it